c/tskit/tables.c

```diff
@@ -5014,6 +5141,9 @@ tsk_table_sorter_run(tsk_table_sorter_t *self, const tsk_bookmark_t *start)
             goto out;
         }
     }
+    if (!skip_individuals) {
+        ret = tsk_table_sorter_sort_individuals(self);
```

Comment:
I know it's redundant, but let's stick the ``if (ret != 0)`` stanza in here. It's easy to miss in diffs if someone updates the code subsequently, and forgets to put it in.

Reply:
Done